db_R_tutorial.Rmd

```diff
@@ -15,15 +15,59 @@ output:
 ---
 
 ```{r setup, warning=FALSE, message=FALSE}
+###################### set-up environment section ################################
+# Set the project path to the root level -
+root.dir = rprojroot::find_rstudio_root_file()
```

Comment:
I have to admit that I have not thought much about possible problems with the working directory when working with RStudio-Projects.
I did some reading on this topic here:
https://gist.github.com/jennybc/362f52446fe1ebc4c49f
https://www.tidyverse.org/blog/2017/12/workflow-vs-script/
https://support.rstudio.com/hc/en-us/articles/200526207-Using-Projects

I guess this never got a problem for me (and I use multiple computers, often share projects with colleagues,...) because I exclusively work with RSTudio-Projects and never just with scripts.
When opening a Rproj-file the working directory is always correctly set and I always work in relative fashion from there.
Therefore I think it is not a problem here either but let's discuss it in our online meeting. 

Did you use the file path stored in "root.dir" somewhere else in the tutorial?

Reply:
Like you I work with project in Rstudio
root.dir is a variable so I'm sure about the path. Specially for package/function that want to write some results in a specific folder.
It's robust with all the OS. So, it's easier when you work with collegues not on the same OS (myself I work on macOS and few times on Linux (Ubuntu))
That's how I work because at the beginning I had some issue with path.
I don't use the variable root.dir somewhere else in the script. It's just to be sure that the user, even if he load the file in a weird folder, can acces/write the correct file.

But don't sweat it, you can use it if you want.

Reply:
I guess you are right. If someone would open the Rmd instead of the RProj-file it could be a problem. Let's discuss on Friday because if we not use the variable later on it would not help a user if he/she runs into problems...
We should put a comment there in any case. Otherwise a reader might be confused about it because it is never used anywhere else.
Is the package "rprojroot" part of the R base installation?

Reply:
Yes, it's in base.